samples/excel/85-preview-apis/pivottable-slicer.yaml

```diff
@@ -0,0 +1,211 @@
+order: 18
+id: excel-pivottable-slicer
+name: Slicer
+description: Adds a slicer to a PivotTable.
+host: EXCEL
+api_set:
+    ExcelApi: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#add-pivot-table").click(() => tryCatch(addPivotTable));
+        $("#add-slicer").click(() => tryCatch(addSlicer));
+        $("#format-slicer").click(() => tryCatch(formatSlicer));
+        $("#apply-style").click(() => tryCatch(applyStyle));
+        $("#add-filters").click(() => tryCatch(addFilters));
+        $("#remove-filters").click(() => tryCatch(removeFilters));
+        $("#remove-slicer").click(() => tryCatch(removeSlicer));
+
+        async function addSlicer() {
+            await Excel.run(async (context) => {
+                const sheet = context.workbook.worksheets.getItem("Pivot");
+                const slicer = sheet.slicers.add("Farm Sales", "Type");
```

Comment:
This is going to be puzzling to beginners with the slice APIs. The first parameter of the slicers.add method looks like it is the name of slice, but it's actually the name of the source. A code comment would help here.